### PR TITLE
(XW-1734) Re-enable Mercury RWA blockedUserOpenDiffPageAndGoBack test

### PR DESCRIPTION
**Pull request by Grunny**
The test is working when run locally and on Jenkins now, so re-enabling
it.

/cc @Wikia/x-wing

### CODE REVIEW

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* MINOR 1 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MINOR [Remove this unused import 'com.wikia.webdriver.common.core.annotations.RelatedIssue'.](https://github.com/Wikia/selenium-tests/blob/a05d7064f4b48c5eac3b53cec8c902fe1063233b/src/test/java/com/wikia/webdriver/testcases/mercurytests/RecentWikiActivityTest.java#L5) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)

File: src/test/java/com/wikia/webdriver/testcases/mercurytests/RecentWikiActivityTest.java [hunk 1]
```diff
@@ -35,8 +34,7 @@ public void mercury_recentWikiActivity_anonOpenDiffPageAndGoBack() {
   /**
    * Access denied when log in
```
Comment (tomasznapieralski) [inline src/test/java/com/wikia/webdriver/testcases/mercurytests/RecentWikiActivityTest.java]: You can also remove that comment :)

Comment (Grunny) [inline src/test/java/com/wikia/webdriver/testcases/mercurytests/RecentWikiActivityTest.java]: Done. :)

**Comment by tomasznapieralski** [comment]
LGTM

**Pull request merged**